profiles/README.MD

```diff
@@ -0,0 +1,7 @@
+
+This is the main repo page for published JSON-LD profiles for xAPI. 
```

Comment:
I think "main repo page" is going to be confusing.

Also, I strongly recommend not lumping actual profiles in with the spec. That's going to make maintainability very bad, both for the spec and the profiles. Really each profile should have its own repo, or possibly if one group is managing multiple closely related profiles maybe a single repo for those.